test/runtests.jl

```diff
@@ -8,7 +8,7 @@ using Sockets: localhost
 using Test
 
 # https://en.wikipedia.org/wiki/Ephemeral_port
-const EPHEMERAL_PORT_RANGE = 49152:65535
+const EPHEMERAL_PORT_RANGE = 49152:60999
```

Comment:
Did you run into trouble here? Maybe we want:
```suggestion
const EPHEMERAL_PORT_RANGE = 32768:60999
```

Reply:
Over engineeered solution I had in another project:
```suggestion
const EPHEMERAL_PORT_RANGE = if Sys.islinux()
    UnitRange(parse.(Int, split(readchomp("/proc/sys/net/ipv4/ip_local_port_range"), '\t'))...)
elseif Sys.isapple()
    UnitRange(parse.(Int, split(readchomp(`sysctl -n net.inet.ip.portrange.first net.inet.ip.portrange.last`),  '\n'))...)
else
    49152:65535  # IANA dynamic and/or private port range start (https://en.wikipedia.org/wiki/Ephemeral_port)
end
```

Reply:
TBH I alredy kinda feel like using a random ephemeral port is overengineered a bit.  I'm just gonna leave it hard coded like this with a note.